src/app/launcher/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts

```diff
@@ -50,10 +90,18 @@ export class ProjectProgressCreateappNextstepComponent implements OnInit, OnDest
         break;
       }
     }
+    if (result) {
+      this.socket.close();
+    }
     return result;
   }
 
   get progress(): Progress[] {
     return this._progress;
   }
+
+  private closeConnections() {
+    this.socket.close();
```

Comment:
Should probably test if the socket is not undefined? For example, if statusLink is not provided then this won't be set.